config/webpack/environment.js

```diff
@@ -1,5 +1,7 @@
 const { environment } = require('@rails/webpacker')
+const erb =  require('./loaders/erb')
```

Comment:
I didn't end up using this. Guess I'll remove it.

Reply:
👍  I guess, this to be removed in another PR?

Reply:
I'll remove it from this one